### ISSUE TITLE
JAWS does not read more than three consecutive repeat letters

### ISSUE THREAD

**Issue by Kathryn-Hackman** [FreedomScientific/VFO-standards-support#636]
### Summary

When reading three or more consecutive letters on a label, JAWS only reads the first three out loud. (for example, if the label text is "YYYY", JAWS will only repeat "Y" three times.) The same will occur for a longer string of Y's, for example "YYYYYYYY" (the letter is still only read three times). However, if that same label text is changed so there are not more than three of one letter in a row (for example, changed to "YYYX"), then each letter is read out loud as expected. This issue is not specific to the letter Y, and happens with lowercase letters as well as uppercase. 

### Expected result

Each letter should be pronounced, for example if there are four Y's as a part of a label ("YYYY") then the letter Y should be read out loud four times.

### Actual result

JAWS read "YYYY" (four Y's) as "YYY" (three Y's)

#### JAWS version and build number
JAWS Version 2022.2202.38 ILM
 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome Version 101.0.4951.67 (Official Build) (64-bit)

**Comment by JAWS-test** [comment]
This applies not only to labels, but in general, e.g. also to text or values in input fields. It seems to be an intended mechanism of JAWS to avoid repetitive output. Can it possibly be deactivated in the settings? I would also consider it a bug

**Comment by Kathryn-Hackman** [comment]
Thank you for your response, I looked through the settings documentation page and I wasn't able to find anything pertaining to repeating letters, however I am pretty new to development with JAWS so I may have missed something.

**Comment by davidengebretson** [comment]
1.  Press JAWSKey+f2 to open Settings Center
  2.  Press control+shift+'d' to open the default settings for all applications
  3.  Type "repeat" (without the quotes) in the search field
  4.  Arrow down to Filter Repeated Characters
     *   Press the space bar to change the value to what you'd like
  5.  Tab to the OK button and press enter to save your changes

**Comment by JeremiahRogersVispero** [comment]
As @davidengebretson shares, this is designed behavior and can be controlled in settings.
